config.py

```diff
@@ -20,3 +20,5 @@
 BOT_IDENTITY = {
     'token': os.environ['COBOT_TOKEN']
 }
+
+DIVERT_TO_PRIVATE = ('help', )
```

Comment:
So this means if I type `corobo help` I'll receive a PM of the help command. Is it possible to point this out to the user with one sentence in that group itself?